pool/web/handler/admin_admins.ml

```diff
@@ -103,15 +103,23 @@ module Access : module type of Helpers.Access = struct
   let read =
     (fun id ->
       let target_id = id |> Uuid.target_of Admin.Id.value in
-      ValidationSet.One (Action.Read, TargetSpec.Id (`Admin, target_id)))
+      ValidationSet.(
+        Or
+          [ One (Action.Read, TargetSpec.Entity `Admin)
```

Comment:
This line shouldn't be needed anymore with guardian.0.1.x

Reply:
Not sure why, but without this line, I cannot access those pages. Same for the update handler below.